source/clear-linux/guides/maintenance/hostname.rst

```diff
@@ -0,0 +1,77 @@
+.. _hostname:
+
+Modify hostname on |CL-ATTR|
+############################
+
+This guide describes how to modify and view the hostname of your 
+|CL-ATTR| system.
+
+
+By default, |CL| installations have a machine generated name, which is a 
+long string of letters and numbers. The generated name is fine for computers 
+but is not human-friendly. Administrators and users will often want to rename 
+their machines with a name that is easier to remember, type, and search 
+for. Renaming a machine also makes it easier to identify, by including 
+meaningful data in the name.  The following examples show human-friendly machine names:
+
+   * *regression-test*
+   * *sally-test-box1*
+   * *az-bldg2-lab*
+
+
+Set your hostname
+*****************
+
+|CL| uses the :command:`hostnamectl` command to display and modify the 
+machine name. :command:`hostnamectl` is part of the **os-core** bundle. 

```

Comment:
@krisdale. I still struggle with the wording of this. Can this sentence be removed, revised? What is the intent in saying that a Linux `command`, `hostnamectl`, is part of ... a bundle? Like we discussed, this is just a general Linux command.